src/CloudGraphsDFG/entities/CGStructure.jl

```diff
@@ -10,17 +10,34 @@ mutable struct User <: AbstractCGNode
     id::Symbol
     name::String
     description::String
-    # labels::Vector{Symbol}
     data::Dict{Symbol, String}
+    createdTimestamp::String
+    lastUpdatedTimestamp::String
+    User(id::Symbol,
```

Comment:
https://docs.julialang.org/en/v1/manual/constructors/#man-inner-constructor-methods-1

> It is good practice to provide as few inner constructor methods as possible: only those taking all arguments explicitly and enforcing essential error checking and transformation. Additional convenience constructor methods, supplying default values or auxiliary transformations, should be provided as outer constructors that call the inner constructors to do the heavy lifting. This separation is typically quite natural.

Reply:
Huh, good note! Thanks, I'll move it outside.